src/app/in-court/hearing-data.service.ts

```diff
@@ -12,10 +12,14 @@ export class HearingDataService {
   public loadHearingDetails(sessionId: string): Observable<any[]> {
     return new Observable<any[]>(observer => {
       this.http.get<any>(`/icp/sessions/${sessionId}`).subscribe(resp => {
-        this.docDataService.getDataFromUrls(resp.documents).subscribe(docs => {
-          observer.next(docs);
-          observer.complete();
-        });
+        observer.next(resp.documents);
+        observer.complete();
```

Comment:
not sure what this is doing

Reply:
Probably not needed anymore, I just copied part of the old code. I'll change that now